pg_stream_copy/protocol.py

```diff
@@ -139,6 +141,15 @@ def build_date(day: date) -> bytes:
     return build_integer(days)
 
 
+def build_timestamp(value: datetime):
+    timestamp_ms = int((value.timestamp() - pg_date_epoch_ts) * 1_000_000)
+    return _build_value(pack('>q', timestamp_ms))
```

Comment:
According to docs https://www.postgresql.org/message-id/d34l6e$284h$1@news.hub.org
```
A timestamp is returned as an 8-byte **big-endian double precision** number of
seconds since POSTGRES_EPOCH_DATE.
```

Reply:
good catch but it was valid up to PG 8.4, then they have changed default into 64-bit integer https://www.postgresql.org/docs/8.4/release-8-4.html
> Use 64-bit integer datetimes by default (Neil Conway)
> Previously this was selected by configure's --enable-integer-datetimes option. To retain the old behavior, build with --disable-integer-datetimes.

Reply:
![Zrzut ekranu z 2019-12-13 21-05-02](https://user-images.githubusercontent.com/8168459/70828941-bd155280-1dec-11ea-89b0-7af43f5d9da3.png)
as you can see postgresql generates the same value for both types of timestamp